packages/suite-desktop/src/modules/coinjoin.ts

```diff
@@ -104,12 +104,16 @@ export const init: Module = ({ mainWindow }) => {
             clientProxyOptions,
         );
 
-        const dispose = () => {
+        const disposeBackends = () => {
```

Comment:
@szymonlesisz What do you think about disposing backends like that?

Or should only be:

```javascript 
        const disposeBackends = () => {
            backends.forEach(b => b.cancel());

            clients.forEach(c => c.disable());
        };

```